pyfolio/timeseries.py

```diff
@@ -512,7 +516,8 @@ def calc_multifactor(returns, factors):
     return results.params
 
 
-def rolling_beta(returns, benchmark_rets, rolling_window=63):
+def rolling_beta(returns, benchmark_rets,
```

Comment:
+1 on using `factor_rets` over `benchmark_rets`. It's more general and in line with academic/industry lingo.